portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -20,35 +20,38 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 import java.util.ArrayList;
+import com.google.gson.Gson;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-
+    private ArrayList<String> comments = new ArrayList<String>();
```

Comment:
I know that the example has it, but note that you can't rely on instance fields in servlets keeping their value. But you'll remove this in the next step, so don't worry about it.